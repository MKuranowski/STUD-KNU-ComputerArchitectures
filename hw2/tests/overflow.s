lui x20, 23782
addi x20, x20, 240
lui x21, 414953
addi x21, x21, 1785
addi x22, x0, 1
slli x22, x22, 31
addi x22, x22, -1
mul x1, x20, x20
mul x2, x21, x21
addi x3, x22, 1
addi x4, x0, 1946
slli x4, x4, 24
addi x5, x0, 1946
srli x5, x5, 24
