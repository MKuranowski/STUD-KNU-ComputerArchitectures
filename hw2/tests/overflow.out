Processor's clock cycles: 16
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 64
x00 = 0
x01 = -2030771968
x02 = -758858191
x03 = -2147483648
x04 = -1711276032
x05 = 0
x06 = 0
x07 = 0
x08 = 0
x09 = 0
x10 = 0
x11 = 0
x12 = 0
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 97411312
x21 = 1699649273
x22 = 2147483647
x23 = 0
x24 = 0
x25 = 0
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = -559038737
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
